Cargo.toml

```diff
@@ -25,7 +25,7 @@ bench-utils = { git = "https://github.com/HorizenOfficial/ginger-lib", branch =
 r1cs-core = { git = "https://github.com/HorizenOfficial/ginger-lib", branch = "algebra_ocl"}
 r1cs-std = { git = "https://github.com/HorizenOfficial/ginger-lib", branch = "algebra_ocl"}
 
-poly-commit = { git = "https://github.com/HorizenLabs/poly-commit", branch = "dev" }
+poly-commit = { git = "https://github.com/HorizenLabs/poly-commit", branch = "labeled_rands" }
```

Comment:
You should sync this with "dlog_segmentation" too

Reply:
Done